Comms.lua

```diff
@@ -103,6 +103,10 @@ function WarpDeplete:OnObjectiveSyncResponse(prefix, message, dist, sender)
   end
 
   self:UpdateObjectivesDisplay()
+  
+  if self.db.profile.timingsEnabled and not self.db.profile.timingsOnlyCompleted then
+    self:UpdateTimings()
```

Comment:
I think I would prefer if this check happened inside the `UpdateTimings` function, as this logic is duplicated in another place as well. That way, from the "outside", only `UpdateTimings` is called, and the logic for what should happen depending on settings is all in one place.